nl_NL/LC_MESSAGES/discordbot.po

```diff
@@ -39,76 +39,76 @@ msgid "MUSICMATCH_MOVIE_COUNT_SINGULAR"
 msgstr "%d Flipnote"
 
 msgid "MUSICMATCH_NO_RESULTS"
-msgstr "I didn't find anything for that, sorry!"
+msgstr "Ik heb niets gevonden voor je, sorry!"
 
 msgid "MUSICMATCH_RESULT_HEADER_GT_MAX"
-msgstr "Top %d results:"
+msgstr "Top %d resultaten:"
 
 msgid "MUSICMATCH_RESULT_HEADER_LT_MAX"
-msgstr "%d result(s):"
+msgstr "%d resultaat(en):"
 
 # SongName by BandName
 msgid "MUSICMATCH_TITLE_BY_ARTIST"
-msgstr "%s by %s"
+msgstr "%s door %s"
 
 msgid "REJECT_ACCOUNT_ALREADY_LINKED"
-msgstr "You've already linked your Sudomemo account!"
+msgstr "Je hebt je sudomemo account al gekoppeld!"
 
 msgid "REJECT_BOOST_TICKET_COOLDOWN"
-msgstr "You've already given a Nitro Booster ticket this month! (%s)"
+msgstr "Je hebt al een nitro booster ticket weggegeven deze maand! (%s)"
 
 msgid "REJECT_FSID_ENOENT"
-msgstr "I did not find a user with that Flipnote Studio ID on Sudomemo."
+msgstr "Ik heb geen gebruiker gevonden met die FSID."
 
 msgid "REJECT_GENERIC_ERROR"
-msgstr "I ran into an issue trying to do that. Sorry!"
+msgstr "Ik ben problemen tegengekomen. Sorry!"
 
 msgid "REJECT_INVALID_FSID"
-msgstr "That is not a valid Flipnote Studio ID!"
+msgstr "Dat is geen geldige FSID!"
 
 msgid "REJECT_INVALID_LINK_TOKEN"
-msgstr "That is not a valid link token! Please visit %s to get started!"
+msgstr "Dat is geen geldige Koppel Token! Klik Hier %s om te starten!"
 
 msgid "REJECT_LINK_TOKEN_ENOENT"
-msgstr "The link token you provided is expired or does not exist."
+msgstr "De link die je hebt gegeven is niet geldig of is verlopen."
 
 msgid "REJECT_NOT_NITRO_BOOSTER"
-msgstr "You need to be a Sudomemo Nitro Booster to use this!"
+msgstr "Je moet een sudomemo nitro booster zijn om dit te doen!"
 
 msgid "REJECT_USER_ENOENT"
-msgstr "I didn't find that user on Sudomemo."
+msgstr "Ik heb die gebruiker niet op sudomemo gevonden."
 
 # "Usage: example_command <fsid>"
 msgid "USAGE"
-msgstr "Usage"
+msgstr "Gebruik"
 
 # ONLY TRANSLATE: "usage", "commands"
 msgid "USAGE_INTRO"
 msgstr ""
-"usage: .sudo ...\n"
+"Gebruik: .sudo ...\n"
 "\n"
-"commands:"
+"commando's:"
```

Comment:
The ' doesn't seem to fit here. I'm not an expert on Dutch however I'm pretty sure it means the same as English, in that it's possessive.

Reply:
well sometimes it could also be comandos but commands is not a word in dutch

Reply:
so i will just call it comandos

Reply:
*commandos